### DISCUSSION TITLE
Convert/port sf (R object) to Geopandas.

### DISCUSSION THREAD

**Discussion by smk5g5** [geopandas/geopandas#3050] [General]
Hi,

I have a sf object in R (on which I have done quite a bit of post-processing) which I need to port/move/analyze in python (just because of the speedup python will provide). I was wondering what would be a good way to move or save sf file such that it can be read by geopandas. I tried saving my sf file as `.shp` using st_write (shape file) but it is not being read by geopandas as it may be missing other files e.g. '.db' that are supposed to be in the folder (as sf by default does not write them). I was wondering what would be the easiest way to move/port/convert sf objects into geopandas for seamless downstream analysis. 

Thank you!

**Comment by martinfleis** [comment]
> I tried saving my sf file as .shp using st_write (shape file) but it is not being read by geopandas as it may be missing other files e.g. '.db' that are supposed to be in the folder (as sf by default does not write them)

I somehow don't believe that `sf` would produce invalid Shapefile. You can try to use any other geospatial format like GPKG, or if you need it to be performant you can go with GeoParquet. Though I am guessing GPKG would be enough. It is in most cases. And it is a one file, so you don't have the issue with loosing side-car files like in the case of a Shapefile.
